crates/ruff_python_formatter/src/pattern/pattern_match_value.rs

```diff
@@ -1,19 +1,16 @@
 use ruff_formatter::{write, Buffer, FormatResult};
 use ruff_python_ast::PatternMatchValue;
 
-use crate::{not_yet_implemented_custom_text, FormatNodeRule, PyFormatter};
+use crate::{AsFormat, FormatNodeRule, PyFormatter};
```

Comment:
Nit: `use crate::prelude::*`. It should allow removing `Buffer`, `FormatResult`, `PyFormatter`, and `AsFormat` (and it makes it easier to change the code in the future without having to add a ton of imports)